sdk/servicebus/ci.yml

```diff
@@ -20,4 +20,4 @@ pr:
 jobs:
 - template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
```

Comment:
You should undo all the changes to this file.

Reply:
You still have file changes in here. You removed the newline at the end of the file. You should be able to use git to just undo all changes.